OpenSSL updates (#139)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/deploy-resource-allocation.adoc

> A C++ based load generator with better efficiency in SSL connections was used to yield the maximum throughput.

Reviewer: Just to avoid the hyphenation issue...:
```suggestion
A load generator based on C++, which has better efficiency in SSL connections, was used to yield the maximum throughput.
```

> The following table lists the approximate requests (averaging 10 KB) that can be served with a single replica of the internal load balancer based on the number of CPU cores. In most cases, ECDSA provides double the performance of a 2K RSA key. Supported curves are secp521r1 (P-521), secp384r1 (P-384), and secp256r1, also known as prime256v1 (P-256).

Reviewer: ```suggestion
The following table shows the approximate requests (averaging 10 KB) that can be served with a single replica of the internal load balancer, based on the number of CPU cores. In most cases, ECDSA provides double the performance of a 2K RSA key. Supported curves are secp521r1 (P-521), secp384r1 (P-384), and secp256r1, also known as prime256v1 (P-256).
```

[%header,cols="4*a"]
|===

> | Key Type | CPU | TLS Without Connection Reuse | TLS With Connection Reuse

Reviewer: ```suggestion
| Key Type | CPU | TLS Without Connection Reuse | TLS with Connection Reuse
```
